bedrock/stories/templates/stories/articles/art-of-engagement.html

```diff
@@ -0,0 +1,79 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "stories/stories-base-article.html" %}
+{% block article_theme %}pink{% endblock %}
+
+
+{% block page_title %}Stories{% endblock %}
+
+{% set featured = true %}
+{% set credits = [
+  {"name": "Ray Fembro", "title": "Product Manager", "image": "img/stories/ray-fembro-credit.png"},
+  {"name": "Ania Safko", "title": "Product Manager", "image": "img/stories/ania-safko-credit.png"},
+  {"name": "Jon Siddoway", "title": "Community Manager", "image": "img/stories/jon-siddoway-credit.png"},
+] %}
+
+{% block article_title %}The Art of Engagement{% endblock %}
+{% block article_subtitle %}
+  Mozilla’s vibrant community gets a direct connection to the people building the products they love.
+{% endblock %}
+
+{% block article_content %}
+<section class="c-stories-article-section">
+  <p>Ray Fembro began by asking for feedback about all things regarding information organization. The project manager then got more specific: what thoughts did people have about tab management?</p>
+  <p>Plenty, as it turned out. Every day for the next two weeks, Fembro signed in to Connect, Mozilla’s new community platform, to listen. He reviewed comments, dissected feedback and otherwise engaged with Mozilla’s user community—one known for its talent, knowledge and an enthusiasm sometimes bordering on fervor.</p>
+  <p>With the launch of Connect, Mozilla has brought that community more directly into conversations with the employees in the midst of building the products they love. The platform acts as a collaborative space where people share product feedback, submit ideas, and have the kind of meaningful discussions that can impact future releases. “We’re opening up new opportunities for us to connect more with our users and make them more a part of product development,” said Product Community Director Jon Siddoway.</p>
```

Comment:
feels like we should definitely have a link to Connect somewhere in here: https://connect.mozilla.org/
@slightlyoffbeat are there UTM params expected for links from the stories blog articles